qulice-maven-plugin/src/main/java/com/qulice/maven/DefaultMavenEnvironment.java

```diff
@@ -238,10 +239,11 @@ public boolean exclude(final String check, final String name) {
 
     @Override
     public Collection<String> excludes(final String checker) {
-        return Collections2.transform(
+        final Collection<String> excludes = Collections2.transform(
```

Comment:
I don't think so. If excludes don't match the checker it returns collection of nulls with size of excludes as was described in the task. I think it is wrong behaviour to return collection of nulls - can affect in the future in some other place again with NPE